src/widgets/ContentEditable.js

```diff
@@ -0,0 +1,314 @@
+import $ from 'jquery';
+import _ from 'lodash';
+import KeyCodes from 'keycodes-enum';
+import TimelineMax from 'TimelineMax';
+import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
+
+const PAD = 2;
+const MODES = {
+  INSERT: 0,
+  OVERWRITE: 1,
+};
+
+export default class ContentEditable extends createjs.Container {
```

Comment:
From thinking this attribute/class through more, it's a niche enough use case that can otherwise be supported in CAM (due to the separation of createjs/canvas for rendering and CAM for a11y) by nesting a single or multiline text input inside another registered DisplayObject to communicate the outer role (e.g. math) and is likely going to require more complexity than is presently in this and the CAM PR, I think we can skip support for this global attribute.

Reply:
removed this global attribute

Reply:
removed